src/sentry_plugins/slack/client.py

```diff
@@ -19,6 +26,6 @@ def request(self, data):
                 path=self.webhook, method="post", data=data, json=False, allow_text=True
             )
         except ApiError as e:
-            # Ignore 404 from slack webhooks
-            if e.code != 404:
+            # Ignore 4XX from slack webhooks
+            if 401 <= e.code <= 404 or e.text not in IGNORABLE_SLACK_ERRORS:
```

Comment:
1) Is every 4XX actually ignorable? or only a subset?
2) Please add a unit test

Reply:
Good point, I'm not 100% sure every 4XX should be included so I dialed it back and added a test.